### PR TITLE
[nctl] NDRS-523: misc enhancements

### PR DESCRIPTION
**Pull request by ghost**
1. Added new nctl-view-* commands to view on chain information.
2. Streamlined node-start command.
3. A few other miscellaneous changes.

### CODE REVIEW

File: utils/nctl/sh/utils/set_aliases.sh [hunk 1]
```diff
@@ -41,12 +41,27 @@ alias nctl-stop='source $NCTL/sh/node/stop.sh'
...
+alias nctl-view-chain-deploy='source $NCTL/sh/views/view_chain_deploy.sh'
 
+# Views: node.
 alias nctl-view-node-log='source $NCTL/sh/views/view_node_log.sh'
-alias nctl-view-node-metrics='source $NCTL/sh/views/view_node_metrics.sh'
 alias nctl-view-node-peers='source $NCTL/sh/views/view_node_peers.sh'
 alias nctl-view-node-status='source $NCTL/sh/views/view_node_status.sh'
 alias nctl-view-node-storage='source $NCTL/sh/views/view_node_storage.sh'
+
+# Views: node metrcs.
```
Comment (linclelinkpart5) [inline utils/nctl/sh/utils/set_aliases.sh:56]: Can't wait to use these!

**Review by linclelinkpart5 — APPROVED** [review]

**Comment by momipsl** [comment]
Cancelled as I have not updated the nctl docs.  Doing that now.

**Pull request closed without merging**